src/hooks/useDashboard.ts

```diff
@@ -133,6 +136,7 @@ const useDahsboard = () => {
     let reservesINDEXLP: TimeSeries[] = [];
     let reservesINDEX: TimeSeries[] = [];
     let reservesUMA: TimeSeries[] = [];
+    let reservesDAO: TimeSeries[] = [];
```

Comment:
```suggestion
    let reservesYAMHOUSE: TimeSeries[] = [];
```

Reply:
lets keep the naming convention related to the token as much as possible, `HOUSE` or `DAOHOUSE` instead of `DAO`